crowbar_framework/Gemfile

```diff
@@ -15,7 +15,7 @@
 # limitations under the License.
 #
 
-source "https://rubygems.org"
+source "http://rubygems.org"
```

Comment:
This is not about encrypting, it's about checking that the server is the one it claims to be.